lib/src/data/report.dart

```diff
@@ -0,0 +1,94 @@
+
+
+import "package:btc_market/data.dart";
+
+/// The type of item, used in reports only for now.  
+enum ItemType {
```

Comment:
Just realized, maybe let's name this `ReportType` since we were thinking of adding specific item (ie, product) types